Use constexpr DoubleToStringConverter singletons in EcmaScriptConverter() and CSSConverter().

MenloDorian opened the pull request:
#### 89e322db73e37f879711ec37d0432248058b164c
<pre>
Use constexpr DoubleToStringConverter singletons in EcmaScriptConverter() and CSSConverter().
<a href="https://bugs.webkit.org/show_bug.cgi?id=242555">https://bugs.webkit.org/show_bug.cgi?id=242555</a>

Reviewed by Darin Adler and Yusuke Suzuki.

Previously, the DoubleToStringConverter singletons were initialized at runtime.  By making
them constexpr, they can be initialize at compile and link time rather than at runtime.

* Source/WTF/wtf/dtoa/double-conversion.cc:
* Source/WTF/wtf/dtoa/double-conversion.h:
(WTF::double_conversion::DoubleToStringConverter::DoubleToStringConverter):

Canonical link: <a href="https://commits.webkit.org/252321@main">https://commits.webkit.org/252321@main</a>
</pre>

Constellation reviewed (approved):
r=me

darinadler reviewed (approved):
Now that these are constexpr, we should find out if moving these to the header is a progression.

Commit:
Use constexpr DoubleToStringConverter singletons in EcmaScriptConverter() and CSSConverter().
https://bugs.webkit.org/show_bug.cgi?id=242555

Reviewed by Darin Adler and Yusuke Suzuki.

Previously, the DoubleToStringConverter singletons were initialized at runtime.  By making
them constexpr, they can be initialize at compile and link time rather than at runtime.

* Source/WTF/wtf/dtoa/double-conversion.cc:
* Source/WTF/wtf/dtoa/double-conversion.h:
(WTF::double_conversion::DoubleToStringConverter::DoubleToStringConverter):

Canonical link: https://commits.webkit.org/252321@main
